src/server/http/urlLookup.js

```diff
@@ -0,0 +1,99 @@
+const crypto = require('crypto');
+
+export default function (server) {
+  async function updateMetadata(urlId, urlDoc) {
+    const client = server.plugins.elasticsearch.client;
```

Comment:
Can these `const client = server.plugins.elasticsearch.client;` lines be moved to a single line within the top-level function?